File: leetcode/1. Two Sum/1._Two_Sum_2.java
public class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] output = new int[2];
        for(int i=0; i<nums.length; i++) {
            int remain = target - nums[i];
            for(int j=i+1; j<nums.length; j++) {
                if(nums[j] == remain) {
                    output[0] = i;
                    output[1] = j;
                    return output;
                }
            }
        }
        return output;
    }
}

File: leetcode/1. Two Sum/1._Two_Sum_2.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        for i in xrange(len(nums)):
            temp = self.findTargetIndex(nums, i, target-nums[i])
            if temp != -1:
                return [i, temp]
        return None
        
    def findTargetIndex(self, nums, skipIndex, target):
        for i in xrange(len(nums)):
            if i == skipIndex:
                continue
            if nums[i] == target:
                return i
        return -1